packages/web-frontend/src/utils/periodGranularities.js

```diff
@@ -115,6 +120,14 @@ export function roundStartEndDates(granularity, startDate = moment(), endDate =
   };
 }
 
+export const formatMomentAsString = (date, granularity, format) =>
+  granularity === 'week' || granularity === 'one_week_at_a_time'
```

Comment:
Let's use the symbolic constants defined in this file instead of `week`/`one_week_at_at_time`